modules/nf-core/pydamage/filter/tests/main.nf.test

```diff
@@ -0,0 +1,46 @@
+
+nextflow_process {
+
+    name "Test Process PYDAMAGE_FILTER"
+    script "../main.nf"
+    process "PYDAMAGE_FILTER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pydamage"
+    tag "pydamage/filter"
+
+    test("test-pydamage-filter") {
```

Comment:
how about a stub test?

Reply:
I've added stubs, tests are however relatively quick to run.

Reply:
Yes, but we want to test that the stub works, it is a seperate section of the code.